conf/config.neon

```diff
@@ -773,6 +773,9 @@ services:
 	-
 		class: PHPStan\Rules\NullsafeCheck
 
+	-
+		class: PHPStan\Rules\Properties\LazyAlwaysUsedClassConstantsExtensionProvider
```

Comment:
Just noticed wrong namespace here as well. `Rules\Constants` will work better, for all the `*AlwaysUsedClassConstantsExtension*` added classes.